src/SFA.DAS.AssessorService.Application.Api/Controllers/StandardServiceController.cs

```diff
@@ -40,7 +33,7 @@ public StandardServiceController(ILogger<StandardServiceController> logger, ISta
         public async Task<IActionResult> StandardServiceGetAllStandardSummaries()
         {
             _logger.LogInformation($@"Get all StandardSummaries from Standard Service");
```

Comment:
Do we need this as it used to return a list of StandardSummary
Think the naming of this method going forward will get confusing

Reply:
Yes, good point.  Will rename.